include/crocoddyl_msgs/whole_body_state_publisher.h

```diff
@@ -151,8 +212,10 @@ class WholeBodyStateRosPublisher {
   Eigen::VectorXd qref_;
   Eigen::VectorXd qfull_;
   Eigen::VectorXd vfull_;
+  Eigen::VectorXd afull_;
   Eigen::VectorXd ufull_;
   bool is_reduced_model_;
+  pinocchio::Inertia temp_Inertia_;
```

Comment:
Let's follow a unique naming style, i.e.,
```suggestion
  pinocchio::Inertia inertia_tmp_;
```